user/serializers.py

```diff
@@ -22,3 +30,36 @@
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.CharField(label=gettext("Email"), write_only=True)
+    password = serializers.CharField(
+        label=gettext("Password"),
+        style={"input_type": "password"},
+        trim_whitespace=False,
+        write_only=True,
+    )
+    token = serializers.CharField(label=gettext("Token"), read_only=True)
```

Comment:
The `AuthTokenSerializer` is correctly implemented to handle user authentication, ensuring that email and password are required and validated.